.github/actions/build-vault/action.yml

```diff
@@ -149,6 +149,12 @@ runs:
         name: ${{ steps.metadata.outputs.artifact-basename }}.zip
         path: out/${{ steps.metadata.outputs.artifact-basename }}.zip
         if-no-files-found: error
+    - name: Copy license file
+      env:
+        LICENSE_DIR: ".release/linux/package/usr/share/doc/${{ inputs.package-name }}"
```

Comment:
For this we probably want to use `inputs.vault-binary-name` instead since in vault-enterprise, the `package-name` will be `vault-enterprise` so the pathing won't be consistent. Also this would be consistent with the dockerfile location of `usr/share/doc/$BIN_NAME`

Reply:
Thanks, Ryan! I didn't know about the script. I will update this to use that instead, I think it makes more sense.